### PR TITLE
PORTALS-1856: set SearchV2 z-index, and fix custom image padding right bug

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/containers/GenericCard.tsx [hunk 1]
```diff
@@ -424,7 +424,7 @@ export default class GenericCard extends React.Component<
       paddingBottom: showFooter || imageFileHandleIdValue ? undefined : '15px',
     }    
     const icon:JSX.Element = <>
-        {imageFileHandleIdValue && <div className="SRC-imageThumbnail" style={{padding: genericCardSchemaDefined.thumbnailRequiresPadding ? '21px' : 'inherit'}}>
+        {imageFileHandleIdValue && <div className="SRC-imageThumbnail" style={{padding: genericCardSchemaDefined.thumbnailRequiresPadding ? '21px' : undefined}}>
```
Comment (jay-hodgson) [inline src/lib/containers/GenericCard.tsx:427]: Setting to an undefined value causes the style not to be overridden (which is the intended behavior in this case)

File: src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.md [hunk 2]
```diff
@@ -54,13 +54,13 @@ Configured to show a table:
       ],
     }}
     searchConfiguration={{
-      searchable: ['assay','name','consortium'],
+      searchable: ['assay','name','consortium', 'study', 'dataType', 'tissue'],
```
Comment (jay-hodgson) [inline src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.md:57]: Changes to the styleguidist demo to cause PORTALS-1856

File: src/lib/style/components/_query-wrapper-plot-nav.scss [hunk 3]
```diff
@@ -95,6 +95,11 @@
     }
   }
 
+  .SearchV2 {
+    // PORTALS-1856: z-index greater than the Show All Graphs button (500)
+    z-index: 600;
```
Comment (jay-hodgson) [inline src/lib/style/components/_query-wrapper-plot-nav.scss:100]: the fix

**Review by emmanmills — APPROVED** [review]
Looks good!

**Pull request merged**